### PR TITLE
Purge 'mapping_3d::'. (#927)

### PR DESCRIPTION
**Pull request by pifon2a**
[Code structure RFC](https://github.com/pifon2a/rfcs/blob/fc64e9fdedd89fca11535b81b2d6f0ce158eafb8/text/0000-code-structure.md)

### CODE REVIEW

File: cartographer/internal/mapping_3d/acceleration_cost_function_3d.h [hunk 1]
```diff
@@ -95,7 +95,7 @@ class AccelerationCostFunction {
   const double second_delta_time_seconds_;
 };
 
-}  // namespace mapping_3d
+}  // namespace mapping
 }  // namespace cartographer
 
 #endif  // CARTOGRAPHER_INTERNAL_MAPPING_3D_ACCELERATION_COST_FUNCTION_H_
```
Comment (gaschler) [inline cartographer/internal/mapping_3d/acceleration_cost_function_3d.h]: Change the namespace to match the file name.

File: cartographer/internal/mapping_3d/acceleration_cost_function_3d.h [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #include "Eigen/Geometry"
 
 namespace cartographer {
-namespace mapping_3d {
+namespace mapping {
 
 // Penalizes differences between IMU data and optimized accelerations.
 class AccelerationCostFunction {
```
Comment (gaschler) [inline cartographer/internal/mapping_3d/acceleration_cost_function_3d.h]: Does this class name need to be updated?

File: cartographer/mapping_3d/scan_matching/real_time_correlative_scan_matcher_3d.h [hunk 3]
```diff
@@ -16,8 +16,8 @@
 
 // A voxel accurate scan matcher, exhaustively evaluating the scan matching
```
Comment (gaschler) [inline cartographer/mapping_3d/scan_matching/real_time_correlative_scan_matcher_3d.h]: drive by: move comment to class?

File: cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d.h [hunk 4]
```diff
@@ -17,8 +17,8 @@
 // This is an implementation of a 3D branch-and-bound algorithm similar to
```
Comment (gaschler) [inline cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d.h]: drive by: move comment to class?

File: cartographer/mapping_3d/scan_matching/translation_delta_cost_functor_3d.h [hunk 5]
```diff
@@ -46,16 +46,16 @@ class TranslationDeltaCostFunctor {
  private:
   // Constructs a new TranslationDeltaCostFunctor from the given
```
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/translation_delta_cost_functor_3d.h]: Done

File: cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d_test.cc [hunk 6]
```diff
@@ -29,15 +29,15 @@
 #include "gtest/gtest.h"
 
 namespace cartographer {
-namespace mapping_3d {
+namespace mapping {
 namespace scan_matching {
 namespace {
 
 class FastCorrelativeScanMatcherTest : public ::testing::Test {
```
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d_test.cc]: Done

File: cartographer/mapping_3d/scan_matching/real_time_correlative_scan_matcher_3d.h (hunk 3 above)
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/real_time_correlative_scan_matcher_3d.h]: Done

File: cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d_test.cc [hunk 7]
```diff
@@ -138,10 +138,10 @@ TEST_F(FastCorrelativeScanMatcherTest, CorrectPoseForMatch) {
   for (int i = 0; i != 20; ++i) {
```
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher_3d_test.cc]: Done

File: cartographer/mapping_3d/scan_matching/rotation_delta_cost_functor_3d.h [hunk 8]
```diff
@@ -57,24 +57,25 @@ class RotationDeltaCostFunctor {
 
  private:
   // Constructs a new RotationDeltaCostFunctor from the given 'target_rotation'.
```
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/rotation_delta_cost_functor_3d.h]: Done

File: cartographer/mapping_3d/scan_matching/precomputation_grid_3d_test.cc [hunk 9]
```diff
@@ -24,17 +24,17 @@
 #include "gmock/gmock.h"
 
 namespace cartographer {
-namespace mapping_3d {
+namespace mapping {
 namespace scan_matching {
 namespace {
 
 TEST(PrecomputedGridGeneratorTest, TestAgainstNaiveAlgorithm) {
```
[the remark this answers is not in the document]
Comment (pifon2a) [reply cartographer/mapping_3d/scan_matching/precomputation_grid_3d_test.cc]: Done

**Pull request merged**